NSS-Day-Cohort-42/rare-API-media-moguls
# Rp post tag

#### Changes Made
1.  added post tag fixture with seed data
1. added posttag.py view file with create list and destroy methods
1. fixed bug in post serializer
​
#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout rp-postTag


3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
server
```
4. Initialize virtual environment, and run the server.
```
pipenv shell
python manage.py runserver
```
5. Test app functionality.
>run python manage.py loaddata posttags
> in postman get post tags by posts with url request post_tags?post_id=1
>try to create a new post tag object in postman. Be sure not to create a post tag relationship that already exists in your database
>test deleting the post tag that you just created
>test getting all posts in postman as well

6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
